website/docs/docs/get-started/privatelink/snowflake-privatelink.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Configuring Snowflake PrivateLink"
+id: snowflake-privatelink
+description: "Configuring PrivateLink for Snowflake"
+sidebar_label: "Snowflake PrivateLink"
+---
+
+## Configure the Snowflake PrivateLink with AWS or Azure
+
+To complete the PrivateLink connection between the dbt and your private networks, first contact dbt Labs support to obtain the AWS or Azure `resource ID`. 
```

Comment:
for our support team, i've been suggesting "dbt Cloud support" because i was influenced by these pages:

- https://docs.getdbt.com/guides/legacy/getting-help#dbt-cloud-support 
- https://docs.getdbt.com/docs/dbt-support

but i actually don't know if we have a standard on this though.

Reply:
If it's what we're doing, I'm on board. I'll keep this in mind going forward